config/files/usr/share/ublue-os/just/60-custom.just

```diff
@@ -28,6 +28,29 @@ set-kargs-hardening-unstable:
     rpm-ostree kargs \
       --append-if-missing="efi=disable_early_pci_dma"
 
+# Add additional boot parameters for hardening (requires reboot)
+remove-kargs-hardening:
+    rpm-ostree kargs \
+      --delete-if-present="init_on_alloc=1" \
+      --delete-if-present="init_on_free=1" \
+      --delete-if-present="slab_nomerge" \
```

Comment:
please refactor both this and the other kargs commands to use a long string stored in a variable so that we don't have to make changes in two places.

something like:

```
kargs_hardened = "string..."
kargs_hardened_unstable = "string..."
```

```
rpm-ostree kargs --append-if-present={{kargs_hardened}}
```


e.g. https://discussion.fedoraproject.org/t/74113

Reply:
Thanks for the tip, hopefully I will get to it tomorrow.